precompile/allowlist/test_allowlist.go

```diff
@@ -539,6 +539,191 @@ func AllowListTests(t testing.TB, module modules.Module) map[string]testutils.Pr
 				require.Equal(t, EnabledRole, GetAllowListStatus(state, contractAddress, TestNoRoleAddr))
 			},
 		},
+		"allowList admin event doesn't log if D fork is not active": {
```

Comment:
Can we adjust these names?

I think a better naming convention is set above with the address of the caller, the role being set, and then adding pre-DUpgrade

For example, this one could be "admin set admin pre-DUpgrade"

Reply:
done